README.md

```
@@ -1,135 +1,259 @@
+
+<p align="center"> <img src="./source/noti.png" width="300" /> </p>
+
+- 알림
+    - 임계치 알림 설정
+    - 정책 적용/해제 알림 설정
+    - 앱 서비스 차단 적용/해제 알림 설정
+    - 선물받은 데이터 알림 설정
+
+### Admin(관리자)
+
+## 로그인
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)